src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1293,6 +1293,13 @@
                 "report_type": "periodic"
             }
         },
+	"BMP": {
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

Mixing tabs and spaces. Please follow existing code's convention.

Reply:
fixed